### ISSUE TITLE
Comparison with mcohen01/node-quickbooks

### ISSUE THREAD

**Issue by AurelienGasser** [apigrate/quickbooks#12]
Upon searching for JS quickbooks libraries, one will stumble opon https://github.com/mcohen01/node-quickbooks as well as this repository.

It would be useful to offer a comparison (in the README?) of this repo v.s. https://github.com/mcohen01/node-quickbooks to inform the user of the differences in approaches, features, and tradeoffs.

**Comment by geoffcorey** [comment]
Yeah the comparison is node-quickbooks is using packages with dependencies that are abandoned https://github.com/mcohen01/node-quickbooks/issues/175

**Comment by gaumeister** [comment]
@geoffcorey @AurelienGasser correct. One of the reasons I created this library was to reduce the risk of dependency pollution. I also have my eye on using native fetch capability available from Node 18 onward, but will hold off for a little while for the sake of backward compatibility.

**Comment by vanta240i** [comment]
> @geoffcorey @AurelienGasser correct. One of the reasons I created this library was to reduce the risk of dependency pollution. I also have my eye on using native fetch capability available from Node 18 onward, but will hold off for a little while for the sake of backward compatibility.

Thank you very much because one of the biggest issues is that

**Comment by triemstr** [comment]
Is this still a better repo than node-quickbooks or what are your thoughts?
